src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -349,6 +350,93 @@ export const exchangeStitchingEnvironment = ({
           },
         },
       },
+      Mutation: {
+        // monkey-patch the existing exchange mutation
+        // to add operations dependent on impulse
+        commerceCreateInquiryOfferOrderWithArtwork: {
```

Comment:
wondering if this needs to be stitched in at all? this looks like a new mutation, which requires data from two services. Should it perhaps be a regular mutation in Metaphysics? it can use `fetch` or something to hit the Exchange GraphQL endpoint (although there's a more `graphql-tools`-y way of hitting a GraphQL endpoint in an ad-hoc way).

Reply:
We used `fetch` to meet a similar challenge in causality's stitching file (maybe `lot.ts` actually) but the issue there is passing the queried fields on to `fetch` assuming we want to return the same exchange type. In causality we solved this by doing querying every field but it might be worth looking into.